### PR TITLE
Rework the story we're telling in the validatorless bootstrap section

### PR DESCRIPTION
**Pull request by tas50**
When we originally wrote this the feature was new so we were letting folks know how to migrate. It's not new at this point and we warn folks not to use it when they bootstrap. Instead we should tell them *why* they shouldn't use it and show them how to move off it. I nuked the layout file that was only used in this single place. There's a ws_ version of it that's not used anywhere that should also go.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: content/install_bootstrap.md [hunk 1]
```diff
@@ -118,17 +118,17 @@ describe how to bootstrap a node using knife.
     just bootstrapped, run the following command:
 
     ``` bash
-    knife client show name_of_node
+    knife client show NAME_OF_NODE
```
Comment (kagarmoe) [inline content/install_bootstrap.md:121]:
```suggestion
    knife client show NODE_NAME
```

File: content/install_bootstrap.md [hunk 2]
```diff
@@ -118,17 +118,17 @@ describe how to bootstrap a node using knife.
...
     ```
 
-    where `name_of_node` is the name of the node that was just
+    where `NAME_OF_NODE` is the name of the node that was just
```
Comment (kagarmoe) [inline content/install_bootstrap.md]:
```suggestion
    where `NODE_NAME` is the name of the node that was just
```

File: content/install_bootstrap.md [hunk 3]
```diff
@@ -118,17 +118,17 @@ describe how to bootstrap a node using knife.
...
     bootstrapped. The Chef Infra Server will return something similar
     to:
 
     ``` bash
     admin:     false
     chef_type: client
-    name:      name_of_node
+    name:      NAME_OF_NODE
```
Comment (kagarmoe) [inline content/install_bootstrap.md]:
```suggestion
    name:      NODE_NAME
```

File: content/install_bootstrap.md (hunk 1 above)
Comment (kagarmoe) [reply content/install_bootstrap.md:121]: Unless there's some reason for using the preposition?

**Comment by tas50** [comment]
Thanks for the cleanup @kagarmoe

**Pull request merged**